spec/forms/collection_edit_form_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'spec_helper'
+include Warden::Test::Helpers
```

Comment:
Without that line it said that it couldn't find the method `login_as`. Is there something else I should include to make that work?